cms/service/ProxyService.py

```diff
@@ -369,6 +377,26 @@ def initialize(self):
         self.enqueue(ProxyOperation(ProxyExecutor.USER_TYPE, users))
         self.enqueue(ProxyOperation(ProxyExecutor.TASK_TYPE, tasks))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/s-maggiolo/cms/file/2607019884/issues/source?bid=3167167&fileBranchId=3309490#l380)